test/functional/test_ivn.py

```diff
@@ -93,23 +98,38 @@ def _start_node(self, node_info):
         fake_node_obj.init()
         fake_node_obj.precheck()
         fake_node_obj.start()
+        fake_node_obj.wait_node_up(timeout=10)
         return fake_node_obj
 
-    def node_config(self, node_name, ns_name, boot_image):
+    def start_node_with_config(self, node_name, ns_name, boot_image,iso):
         fake_node = None
         fake_node = copy.deepcopy(FakeConfig().get_node_info())
         fake_node['name'] = node_name
         fake_node['namespace'] = ns_name
         fake_node['compute']['storage_backend'][0]["drives"][0]["file"] = boot_image
+        fake_node['compute']["cdrom"]=iso
         fake_node["compute"]["networks"][0]["port_forward"] = [{"outside": 8022, "inside": 22, "protocal": "tcp"}]
+        if "test0" in node_name:
+            print "test0"
+            fake_node["compute"]["networks"].append({"device": "e1000", "network_mode": "bridge", "network_name": "br0", "mac": "00:60:16:93:b9:1d"})
+        if "test1" in node_name:
+            print "test1"
+            fake_node["compute"]["networks"].append({"device": "e1000", "network_mode": "bridge", "network_name": "br0", "mac": "00:60:16:93:b9:1a"})
         fake_node_up = self._start_node(fake_node)
         return fake_node_up
 
     def client_ssh(self, ns_ip):
-        ssh = sshclient.SSH(host=ns_ip, username="root", password="root", port=8022)
-        ssh.wait_for_host_up()
-        status, output = ssh.exec_command("ls")
-        return output
+        ssh = sshclient.SSH(host=ns_ip, username="ubuntu", password="passw0rd!!")
```

Comment:
would suggest to put these guest OS info together and place them use a macro instead of hard-code.